gradle/system-test.gradle

```diff
@@ -19,7 +19,9 @@ dependencies {
     systemTestImplementation group: 'io.rest-assured', name: 'rest-assured', version: '5.3.1'
     systemTestImplementation group: 'org.mockito', name: 'mockito-core', version: '5.5.0'
     systemTestImplementation group: 'org.mockito', name: 'mockito-inline', version: '5.2.0'
-
+    systemTestImplementation 'org.springframework.boot:spring-boot-testcontainers'
```

Comment:
Could you please add the version of the dependencies?

Reply:
done